plugins/ranking-system

```diff
@@ -0,0 +1,4 @@
+repository=https://github.com/Landathradon/Ranking-System.git
+commit=db5d0939aab6cd12c2ec61d19ddf470dcad1936b
+warning=This plugin submits the name of your clan and your IP address to a 3rd party website not controlled or verified by the RuneLite Developers.
+authors=Lore Bot
```

Comment:
Who is this? I don't see any github user named  `Lore Bot`.

Reply:
This is me, I thought it was my runescape name I'd put there but I can change it back if thats the wrong way of doing things